src/Symfony/Bundle/TwigBundle/DependencyInjection/Compiler/ExtensionPass.php

```diff
@@ -41,6 +42,17 @@ public function process(ContainerBuilder $container)
 
         if ($container->has('fragment.handler')) {
             $container->getDefinition('twig.extension.httpkernel')->addTag('twig.extension');
+
+            // inject Twig in the hinclude service if Twig is the only registered templating engine
+            if (
+                !$container->hasParameter('templating.engines')
+                || array('twig') == $container->getParameter('templating.engines')
+            ) {
+                $container->getDefinition('fragment.renderer.hinclude')
+                    ->addTag('kernel.fragment_renderer', array('alias' => 'hinclude'))
```

Comment:
That's one of my goal for 2.7, being able to use Twig without the overhead of the Templating component. And if you have a look at my latest PRs, you will see that many of them try to go into that direction.